flows.yaml

```diff
@@ -1601,6 +1601,8 @@ integrations:
                     - https://www.googleapis.com/auth/admin.directory.user.readonly
             workspace-users:
                 runs: every hour
+                input: Metadata
```

Comment:
at some point we should rename input to metadata, it's not really clear unless what input is unless the model name is correct

Reply:
Yeah, agreed